gulpfile.js

```diff
@@ -51,6 +52,49 @@ const { launch, connect } = require('hadouken-js-adapter');
 		console.log(`[${new Date().toLocaleTimeString()}] ${chalk[color][bgcolor](msg)}.`);
 	}
 
+	/** 
+	* Validates the current node version against supported node versions specified in this file
+	* Returns true of false if current node version is valid
```

Comment:
```suggestion
	* Returns boolean indicating whether current node version is valid
```

Or you could do `Returns boolean if and only if the current node version is valid`

Or maybe `Returns true if the current node is valid and false otherwise`